tests/net_dynamic_hb.rs

```diff
@@ -170,7 +170,7 @@ fn do_drop_and_readd(cfg: TestConfig) {
     loop {
         let (node_id, step) = net.crank_expect();
         if !net[node_id].is_faulty() {
-            // Verify that contributions from all participants of the batch are present.
+            // Verify that contributions from all correct participants of the batch are present.
```

Comment:
But we can't expect that either: Even correct participants may fail to get their contribution accepted, if their messages are delayed.